src/sass/disability-benefits.scss

```diff
@@ -573,3 +561,34 @@ p.first-of-type {
 .claims-files-cancel {
   margin-left: 32px;
 }
+
+.item-title-container {
+  float: left;
+  width: 63%;
+  @media (max-width: $small-screen) {
+    float: none;
+    width: 100%;
+  }
+}
+.view-details-button {
+  float: right;
+  margin: 0;
+  @media (max-width: $small-screen) {
+    float: none;
+    width: 100%;
+    margin: 0.5em 0;
+  }
+}
+
+.tab-li {
+  @media (max-width: $small-screen) {
+    width: 33.33%;
```

Comment:
Gotcha, yeah right now is not 100% perfect. You can see maybe a pixel is off on the right of each tab.